docs/samples/index.md

```diff
@@ -0,0 +1,24 @@
+<!--
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+-->
+
+# Ingress APISIX Use Examples
+
+This is an index page about use examples of Ingress APISIX. Read them if you're trying to learn Ingress APISIX.
+
+* [Proxy the httpbin service](#proxy-the-httpbin-service).
```

Comment:
Should be 
[Proxy the httpbin service](./proxy-the-httpbin-service.md)

Reply:
this way is better:

![image](https://user-images.githubusercontent.com/6814606/104556010-c620c500-5679-11eb-9610-a16cf3e8f4d8.png)

```suggestion
[Proxy the httpbin service](./proxy-the-httpbin-service.md)
```

Reply:
Get it .

Reply:
Changed.